code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -348,6 +348,22 @@
 		N.hal_screwyhud = SCREWYHUD_NONE
 	..()
 
+/datum/reagent/medicine/system_cleaner
+	name = "System Cleaner"
+	description = "Neutralizes harmful chemical compounds inside synthetic systems."
+	reagent_state = LIQUID
+	color = "#F1C40F"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	process_flags = SYNTHETIC
+
+/datum/reagent/medicine/system_cleaner/on_mob_life(mob/living/M)
+	M.adjustToxLoss(-2*REM, 0)
```

Comment:
aren't both androids and IPCs immune to tox damage? what's the point of this chem?